src/index.js

```diff
@@ -110,13 +110,23 @@ export default function onClickOutsideHOC(WrappedComponent, config) {
       }
 
       this.componentNode = findDOMNode(this.getInstance());
-      // return early so we dont initiate onClickOutside
+      // return early so we don't initiate onClickOutside
       if (this.props.disableOnClickOutside) return;
       this.enableOnClickOutside();
     }
 
-    componentDidUpdate() {
+    componentDidUpdate({ disableOnClickOutside: prevDisableOnClickOutside }) {
+      const { disableOnClickOutside } = this.props;
```

Comment:
it'll break on this, too. 
```suggestion
      var disableOnClickOutside = this.props.disableOnClickOutside;
```